source/sdk/structs/VectorList.ooc

```diff
@@ -15,14 +15,11 @@
 * along with this software. If not, see <http://www.gnu.org/licenses/>.
 */
 
-import structs/Vector
+import structs/[Vector, List]
```

Comment:
While you're at it, add `Vector` to `sdk.use` just as you did for `List`, and you can remove this entire line.